docs/source/user_guide/memory.rst

```diff
@@ -198,3 +198,51 @@ Be sure to do this before any other CuPy operations.
 
    # Disable memory pool for pinned memory (CPU).
    cupy.cuda.set_pinned_memory_allocator(None)
+
+
+Unified memory programming (UMP) support (**experimental!**)
```

Comment:
Could you change the title to make it clear that the feature is unrelated to managed memory (which is often referred as [Unified Memory](https://developer.nvidia.com/blog/unified-memory-cuda-beginners/))?
This is the doc often referred by, so I'd avoid confusing majority of users who don't own G+H 😅

Reply:
This was purposely changed to reflect the importance of including managed memory here, see the discussion with Rong [starting here](https://github.com/cupy/cupy/pull/8442#issuecomment-2374444257), and the linked devblog. From CUDA Programming Model's perspective (and therefore CuPy), it does not matter if we are using 
- malloc on G+H
- cudaMallocManaged on G+H
- cudaMallocManaged on x84-64

everything here (after we fix a few minor things in a subsequent PR) will work. The performance characteristics may be different, but it is a separate discussion.

Reply:
> everything here (after we fix a few minor things in a subsequent PR) will work

The needed effort is tracked in #8711.